main.tf

```diff
@@ -89,6 +89,14 @@ resource "aws_s3_bucket" "s3_bucket" {
   bucket = "${var.name}"
   acl    = "${var.acl}"
 
+  cors_rule {
```

Comment:
You may also want to include this block here: https://github.com/UKHomeOffice/acp-tf-s3/blob/v0.1.8/main.tf#L134

There's multiple `aws_s3_bucket` definitions with a count depending on what options are passed in.

Reply:
Good point, added